<?xml version="1.0"?>
<launch>

	<!-- upload joint configuration -->
	<rosparam command="load" ns="/script_server/arm" file="$(find cob_silia)/config/arm_joint_configurations.yaml"/>

</launch>
